### ISSUE TITLE
NotImplementedError `sphinx.writers.latex.LaTeXTranslator` departing unknown node type `RailroadDiagramNode`

### ISSUE THREAD

**Issue by r0qs** [taminomara/sphinx-a4doc#16]
Hi, apparently, since release `1.4.0`, the latex build has stopped working with the following error:

```
Running Sphinx v5.3.0
loading translations [tr]... done
making output directory... done
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [latex]: all documents
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
processing solidity.tex... index introduction-to-smart-contracts installing-solidity solidity-by-example layout-of-source-files structure-of-a-contract types units-and-global-variables control-structures contracts assembly cheatsheet grammar using-the-compiler analysing-compilation-output ir-breaking-changes internals/layout_in_storage internals/layout_in_memory internals/layout_in_calldata internals/variable_cleanup internals/source_mappings internals/optimizer metadata abi-spec 050-breaking-changes 060-breaking-changes 070-breaking-changes 080-breaking-changes natspec-format security-considerations smtchecker resources path-resolution yul style-guide common-patterns bugs contributing brand-guide language-influences 
resolving references...
/home/docs/checkouts/readthedocs.org/user_builds/solidity-docs-tr-turkish/checkouts/v0.8.15/docs/introduction-to-smart-contracts.rst:196: WARNING: undefined label: 'özel bir global değişken <special-variables-functions>`dir. ``msg.sender`'
/home/docs/checkouts/readthedocs.org/user_builds/solidity-docs-tr-turkish/checkouts/v0.8.15/docs/introduction-to-smart-contracts.rst:212: WARNING: undefined label: 'hatalar'
/home/docs/checkouts/readthedocs.org/user_builds/solidity-docs-tr-turkish/checkouts/v0.8.15/docs/units-and-global-variables.rst:320: WARNING: undefined label: 'address`e gönderir ve yürütür. ``selfdestruct`'
/home/docs/checkouts/readthedocs.org/user_builds/solidity-docs-tr-turkish/checkouts/v0.8.15/docs/internals/optimizer.rst:305: WARNING: undefined label: 'var-decl-initializer'
/home/docs/checkouts/readthedocs.org/user_builds/solidity-docs-tr-turkish/checkouts/v0.8.15/docs/common-patterns.rst:13: WARNING: undefined label: 'güvenlikle i̇lgili değerlendirmeler'
done
writing... /home/docs/checkouts/readthedocs.org/user_builds/solidity-docs-tr-turkish/checkouts/v0.8.15/docs/grammar/SolidityParser.g4:: WARNING: unknown node type: <RailroadDiagramNode: >
failed

Exception occurred:
  File "/home/docs/checkouts/readthedocs.org/user_builds/solidity-docs-tr-turkish/envs/v0.8.15/lib/python3.7/site-packages/docutils/nodes.py", line 2058, in unknown_departure
    % (self.__class__, node.__class__.__name__))
NotImplementedError: <class 'sphinx.writers.latex.LaTeXTranslator'> departing unknown node type: RailroadDiagramNode
The full traceback has been saved in /tmp/sphinx-err-pn9u4xxa.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
```

The problem seems to be related to this change https://github.com/taminomara/sphinx-a4doc/commit/169ca4384e095e752b34bdf95b9f7c151e9e5abb. If I downgrade `sphinx-a4doc` to version `1.3.0`, it works again.

**Comment by taminomara** [comment]
I had no idea latex build worked... that was an accident!

Anyways, I've pushed experimental support for latex in [c1e8c83](https://github.com/taminomara/sphinx-a4doc/commit/c1e8c83f50ee2f429ec55db163cd1d90c1480b8d), can you check if it works for you so I can make a release?

**Comment by taminomara** [comment]
NVM ran solidity docs build and it works now. So, issue fixed in 1.6.0.

**Comment by r0qs** [comment]
Thanks :)
